### ISSUE TITLE
[Screen Reader - Open and close New Tab]: Screen reader does not announce position number for the tabs navigating via "Ctrl+Tab" or "Ctrl+Shift+Tab" key.

### ISSUE THREAD

**Issue by Chandresh1007** [microsoft/terminal#8474]
**User Experience:**
If the correct  information and relationships are not provided, it would be difficult for people with different disabilities as the information and relationships that are implied by visual or auditory formatting will not be preserved when the presentation format changes.
Having these structures and these relationships programmatically determined or available in text ensures that information important for comprehension will be perceivable to all. 

**Test Environment:**
OS: Win 10 2004 Build (20262.1010)
App: Windows Terminal
App version: 1.4.3243.0

**Repro Steps:**
1. Open Windows terminal app.
2. Launch Narrator.
3. Press "Ctrl+Shift+T" to open multiple tabs.
4. Now press "Ctrl+Tab" or "Ctrl+Shift+Tab" key to navigate between tabs.
5. Observe the issue.

**Actual Result:**
Screen reader **does not announce position number for the tabs** navigating via "Ctrl+Tab" or "Ctrl+Shift+Tab" key.

**Expected Result:**
Screen reader should announce the position number for the tabs navigating via "Ctrl+Tab" or "Ctrl+Shift+Tab" key.
For e.g. **If total 5 tabs are present and currently the focus is at 3rd tab then narrator should announce it as "3of 5".**

**Note:** Same issue repro with NVDA.

**Observation:**  In scan mode narrator announces position number.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7B54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4%7D&cid=a7f80d30-34d6-4e03-af19-fa3c97a5bf31

[MAS1.3.1_Position no does not get announced for tabs.zip](https://github.com/microsoft/terminal/files/5635199/MAS1.3.1_Position.no.does.not.get.announced.for.tabs.zip)

![Tab item_code snippet](https://user-images.githubusercontent.com/53220700/100993542-8b584700-357b-11eb-8904-555315469b07.png)

**Comment by zadjii-msft** [comment]
I'm guessing that this is the same thing as #8464, where we provide a little bit of metadata in the message. /cc @carlos-zamora

**Comment by akash07k** [comment]
@zadjii-msft 
Any update on this issue?

**Comment by zadjii-msft** [comment]
@akash07k @carlos-zamora is the one who's currently working on our accessibility bugs.

Carlos, do browsers do this? If so, then I'm betting this is also something that the WinUI team should implement themselves, like #8464

**Comment by akash07k** [comment]
Thanks @zadjii-msft 
@carlos-zamora I'd love to know the progress on this issue if possible. :-)


> @akash07k @carlos-zamora is the one who's currently working on our accessibility bugs.
> 
> Carlos, do browsers do this? If so, then I'm betting this is also something that the WinUI team should implement themselves, like #8464

Ya, if WinUI team implements it then it will be fixed for many more apps too and it would be win-win for all.
I'd be investigating this issue in more detail tonight.

**Comment by carlos-zamora** [comment]
I have good news and bad news.
Good news:
- tested this on v1.7.501.0 (preview selfhost) and v1.5.10411.0 (stable public release). It got fixed (probably by WinUI)

Bad news:
- seeing a regression on v1.7.501.0. Tab names are not being read out by the screen reader. Submitting a separate issue for that.

**Comment by carlos-zamora** [comment]
@Chandresh1007 since this is fixed now, could you close the bug? Thanks.

**Comment by DHowett** [comment]
Er, I tried this in 1.7 and didn’t hear an announcement for the tab index. Maybe the tab switcher interferes?

**Comment by akash07k** [comment]
@carlos-zamora 
Thanks for confirming,
Can you please let me know that where can I find that selfhosted build? is it publicly available or not?
And, please give the issue link for "tab names not being read" Would love to track it's progress.
> I have good news and bad news.
> Good news:
> 
> * tested this on v1.7.501.0 (preview selfhost) and v1.5.10411.0 (stable public release). It got fixed (probably by WinUI)
> 
> Bad news:
> 
> * seeing a regression on v1.7.501.0. Tab names are not being read out by the screen reader. Submitting a separate issue for that.

**Comment by carlos-zamora** [comment]
Apologies. I was in the process of writing a comment then got distracted with something else. The tab switcher actually "fixes" this functionality. However, when the tab switcher is not enabled, this bug occurs. So it's still valid, I think?

The default functionality setting includes the tab switcher though. That's the only reason I can think of disqualifying it, but that's not a good enough reason, I think.

**Comment by Chandresh1007** [comment]
Issue is fixed. Now the screen reader is announcing the position number along with its name and state. Verified on following Test Environment:
OS: Win 10 2004 Build (21301.1010)
App: Windows Terminal
App version: 1.5.10411.0
Hence, closing the bug.

**Comment by Priyanka-Chauhan123** [comment]
Closing the bug as issue has been fixed.
